docs/sources/mimir/visualize/grafana.md

```diff
@@ -0,0 +1,40 @@
+---
+description: Visualize your metric data with Grafana
+title: Visualize metric data
+menuTitle: Visualize data
+weight: 45
+keywords:
+  - visualize
+  - grafana
+  - dashboards
+---
+
+# Visualize metric data
+
+[Grafana](/grafana/download/) has built-in support for Mimir through the Prometheus data source type.
+
+## Before you begin
+
+1. Log in to your Grafana instance.
+   If this is your first time running Grafana,
+   the username and password are both `admin`.
+1. From within Grafana, go to `Connections` > `Add new connection`.
+1. Search for Prometheus, and select **Create a Prometheus data source**.
+1. In the **HTTP** > **Prometheus server URL** field, point to the proxy in front of Mimir (such as Nginx or the GEM gateway) rather than to one component of Mimir.
+
+   - When running locally or with Docker using port mapping,
+     the address is likely `http://localhost:9090`.
+
+   - When running with docker-compose or Kubernetes,
+     the address is likely `http://prometheus:9090`.
```

Comment:
I think we need to talk about this in terms of the tool used to install Mimir (Helm, Jsonnet etc.) rather than the environment Mimir is running in (local or Kubernetes), as each installation method has different configuration, and that then determines the URL.

For Helm, based on https://grafana.com/docs/helm-charts/mimir-distributed/latest/get-started-helm-charts/#start-grafana-in-kubernetes-and-query-metrics, the default URL to use would be `http://mimir-nginx.<Mimir namespace>.svc/prometheus`.

For Jsonnet/Tanka, we don't install a proxy automatically (probably something we could improve). 

If you're running the binary locally in monolithic mode, you can use `http://localhost:<Mimir port>`. If you're running microservices, then you need a proxy.

Reply:
Please have another look.